client/my-sites/stats/stats-date-picker/date-label-drill/index.tsx

```diff
@@ -0,0 +1,42 @@
+import page from '@automattic/calypso-router';
+import { Icon, levelUp } from '@wordpress/icons';
+import { useState, useEffect, ReactNode } from 'react';
+import { appendQueryStringForRedirection } from '../../utils';
+import './index.scss';
+
+interface QueryParams {
+	drilldown?: string;
+}
+
+interface DateLabelDrillProps {
+	query?: QueryParams;
+	children?: ReactNode;
+}
+
+const DateLabelDrill = ( { query, children }: DateLabelDrillProps ) => {
+	const [ isAnimated, setIsAnimated ] = useState( false );
+
+	useEffect( () => {
+		setIsAnimated( true );
+
+		const params = { ...query };
+		delete params.drilldown;
+		const url = appendQueryStringForRedirection( window.location.pathname, params );
+		window.history.replaceState( params, '', url );
+
+		page.replace( url, null, false, false );
```

Comment:
Remove the `drilldown` query parameter from the URL to prevent unexpected go-backs from being shared by direct paste.